### PR TITLE
Use hash commit to tag che-machine-exec image.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### Referenced issue:
Needed for https://github.com/eclipse/che/issues/12780.
Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### CODE REVIEW

File: assembly/etc/che-plugin.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ endpoints:
       discoverable: false
 containers:
  - name: che-machine-exec
-   image: eclipse/che-machine-exec
+   image: aandrienko/che-machine-exec:latest
```
Comment (benoitf) [inline assembly/etc/che-plugin.yaml]: hello, why is aandrienko ?

Comment (AndrienkoAleksandr) [reply assembly/etc/che-plugin.yaml]: Yes, it's was not needed changes. Thanks.

File: build.sh [hunk 2]
```diff
@@ -7,7 +7,10 @@ NC='\033[0m'
 DIR=$(cd "$(dirname "$0")"; pwd)
 
 printf "${BLUE}Building service (docker image)${NC}\n"
-docker build -t eclipse/che-machine-exec .
+
+docker build --no-cache -t eclipse/che-machine-exec:latest .
+GIT_HASH=$(git rev-parse --short=7 HEAD)
+docker tag eclipse/che-machine-exec:latest aandrienko/che-machine-exec:${GIT_HASH}
```
Comment (benoitf) [inline build.sh]: there is still aandrienko for new image with hash

Comment (AndrienkoAleksandr) [reply build.sh]: Thanks. Done.

**Comment by AndrienkoAleksandr** [comment]
We decided to use latest image for now. When we will want to provide breaking changes we will think about stable and next tags.

**Pull request closed without merging**